sdk/web-pubsub/web-pubsub-client-protobuf/src/webPubSubProtobufReliableProtocol.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { WebPubSubMessage, WebPubSubClientProtocol } from "@azure/web-pubsub-client";
+
+import { createClientLogger } from "@azure/logger";
+
+import { WebPubSubProtobufProtocolBase } from "./webPubSubProtobufProtocolBase";
+
+/**
+ * The \@azure\/logger configuration for this package.
+ */
+export const logger = createClientLogger("web-pubsub-client");
```

Comment:
you should have one logger per package, not multiple instances in different source files. Can pull this out into a new module